pkg/util/mount/nsenter_mount.go

```diff
@@ -176,10 +176,11 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	exec := exec.New()
 	out, err := exec.Command(nsenterPath, args...).CombinedOutput()
 	if err != nil {
-		glog.Errorf("Failed to nsenter mount, return file doesn't exist: %v", err)
-		// If the command itself is correct, then if we encountered error
-		// then most likely this means that the directory does not exist.
-		return true, os.ErrNotExist
+		glog.V(2).Infof("Failed findmnt command: %v", err)
+		// Different operating systems behave differently for paths which are not mount points.
+		// On older versions (e.g. 2.20.1) we'd get error, on newer ones (e.g. 2.26.2) we'd get "/".
+		// It's safer to assume that it's not a mount point.
+		return true, nil
```

Comment:
There's no such thing as "exact error". The only signal we get is that bash command exits with 1.

btw. this used to be like this earlier (see https://github.com/kubernetes/kubernetes/pull/13446). For reasons I don't remember I changed it to few months ago to treat errors as `os.ErrNotExists`, but apparently didn't test it properly at that time.

We can consider checking version of `findmnt` or `util-linux` package, but that might be an overkill (how to do it, anyway?)